docker/generic/start_proxy.py

```diff
@@ -340,6 +340,13 @@ def make_argparser():
         Only works when --cors_preset is in use. Enable the CORS header
         Access-Control-Allow-Credentials. By default, this header is disabled.
         ''')
+    parser.add_argument(
+        '--cors_max_age',
+        default=1728000,
```

Comment:
ACK. But do you mean the other way around? I found all the default values are kept in python script and leave empty for the golang side? I changed my implementation to follow this pattern.

Reply:
No, I think all default values are kept on golang side. This is to support Cloud API Gateway. They do not use the python script, but they use golang. It will break them if the default value is not in golang.

There may be some legacy fields with default value on python side, but all newer fields have it on golang side. Or you can do both (?)

Reply:
I c. I guess because I found all the cors related flags have default values in python but not in golang. Probably these are the legacy fields.

I set the default value to 480h for both python and go.